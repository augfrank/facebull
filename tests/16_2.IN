M0   C0   C2  2276
M1   C0   C3  2336
M2   C0   C4  2151
M3   C0   C13  2615
M4   C1   C6  2249
M5   C1   C9  2283
M6   C1   C15  2478
M7   C2   C6  2637
M8   C2   C7  2500
M9   C2   C9  2241
M10   C2   C11  2572
M11   C2   C15  2733
M12   C3   C2  2233
M13   C3   C5  2171
M14   C3   C10  2765
M15   C3   C12  2880
M16   C4   C1  2852
M17   C4   C3  2065
M18   C4   C5  2263
M19   C4   C7  2317
M20   C4   C8  2607
M21   C4   C10  2600
M22   C4   C11  2402
M23   C4   C13  2265
M24   C5   C4  2550
M25   C5   C6  2219
M26   C5   C8  2942
M27   C5   C10  2305
M28   C6   C1  2871
M29   C6   C2  2291
M30   C6   C3  2565
M31   C6   C4  2377
M32   C6   C10  2649
M33   C6   C11  2356
M34   C7   C3  2229
M35   C7   C5  2638
M36   C7   C8  2564
M37   C7   C13  2940
M38   C8   C2  2766
M39   C8   C5  2518
M40   C8   C15  2253
M41   C9   C1  2617
M42   C9   C4  2146
M43   C9   C7  2241
M44   C9   C13  2748
M45   C10   C0  2454
M46   C10   C1  2595
M47   C10   C5  2678
M48   C10   C9  2092
M49   C10   C13  2382
M50   C11   C4  2629
M51   C11   C5  2172
M52   C11   C8  2420
M53   C11   C10  2557
M54   C11   C14  2497
M55   C12   C5  2235
M56   C12   C7  2548
M57   C13   C1  2678
M58   C13   C5  2713
M59   C13   C7  2555
M60   C14   C10  2239
M61   C14   C12  2774
M62   C15   C4  2535
M63   C15   C11  2568
M64   C15   C12  2856
