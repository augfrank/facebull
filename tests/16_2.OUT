38193
3 5 8 12 17 25 33 36 40 42 45 54 55 57 60 64
